concordium-consensus/src/Concordium/KonsensusV1/Consensus/CatchUp.hs

```diff
@@ -13,3 +54,490 @@ module Concordium.KonsensusV1.Consensus.CatchUp where
 -- Catchup responses must bypass the deduplication layer,
 -- hence if we are catching up from a peer then their responses should
 -- not be deduplicated.
+
+{-
+A node N is up to date with a peer P if:
+    * N considers P's last finalized block to be finalized.
+    * The round of N's highest certified block is at least the round of P's highest certified block.
+    * N's current round (round(N)) is at least P's current round.
+    * N's current epoch is at least P's current epoch.
+    * N has all timeout signatures on round(N) that P has, and considers them valid if they are.
+    * N has all quorum signatures on round(N) that P has, and considers them valid if they are.
+    * If P has only one block in round(N), then N also has that block.
+
+If P is honest and considers a timeout signature on round(N) valid, then
```

Comment:
then what?

Reply:
I have removed this as I think the notes are not so useful any more.